diagrams/EnvireMars_Activity_Diagram.di

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file? This are not the sources for the diagrams. The sources for the activity diagram have not been included...